clayview/src/main/java/com/linecorp/clay/drawable/PathSelectDrawable.kt

```diff
@@ -19,10 +19,12 @@ import com.linecorp.clay.view.DrawingPath
  * Path selection drawable, if highlight, it draws clear inside the path and draw mask outside
  * If not highlight, it just draw the path
  */
-internal class PathSelectDrawable(var drawingPath: DrawingPath, @ColorInt maskColor: Int) : Drawable() {
+internal class PathSelectDrawable(var drawingPath: DrawingPath,
```

Comment:
It looks like this class header formatting is a little different from the guidelines.
https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting

Reply:
yes, i will update it later